source/core_mqtt.c

```diff
@@ -882,8 +882,8 @@ static MQTTStatus_t receivePacket( const MQTTContext_t * pContext,
         if( bytesReceived == ( int32_t ) bytesToReceive )
         {
             /* Receive successful, bytesReceived == bytesToReceive. */
-            LogInfo( ( "Packet received. ReceivedBytes=%ld.",
-                       ( long int ) bytesReceived ) );
+            LogDebug( ( "Packet received. ReceivedBytes=%ld.",
```

Comment:
Should this be debug? A failed receive sounds like something more important.

Reply:
This line is for a successful receive.